atom/common/api/atom_api_crash_reporter.cc

```diff
@@ -31,21 +31,26 @@ struct Converter<CrashReporter::UploadReportResult> {
 
 namespace {
 
-void SetExtraParameter(const std::string& key, mate::Arguments* args) {
-  std::string value;
-  if (args->GetNext(&value))
-    CrashReporter::GetInstance()->SetExtraParameter(key, value);
-  else
-    CrashReporter::GetInstance()->RemoveExtraParameter(key);
+void SetExtraParameter(const std::string& key, const std::string& value) {
+  CrashReporter::GetInstance()->SetExtraParameter(key, value);
 }
 
+void RemoveExtraParameter(const std::string& key) {
+  CrashReporter::GetInstance()->RemoveExtraParameter(key);
+}
+
+std::map<std::string, std::string> GetParameters() {
```

Comment:
CI tells me that `non-member functions cannot have 'const' qualifier`, why is that? i've left it out for now bc otherwise tests are 🚫

Reply:
My bad, I didn't notice this was a nonmember function.

The getter member functions should be const to indicate that they don't change the object, but since nonmember functions have no object to refer to, of course const doesn't make sense there.